BookEcommerce.Models/DAL/Interfaces/ICartRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using BookEcommerce.Models.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookEcommerce.Models.DAL.Interfaces
+{
+    public interface ICartRepository : IRepository<Cart>
+    {
+        Task<List<Cart>> GetCart(string id);
```

Comment:
GetCarts if you want to get a List. But I don't clear about this thing. One Id has many carts. And I think we should getUserCart and use UserGuid right?